nautobot/ipam/forms.py

```diff
@@ -704,7 +713,7 @@ def __init__(self, *args, **kwargs):
                 self.initial["primary_for_parent"] = True
 
     def clean(self):
-        super().clean()
+        self.cleaned_data = super().clean()
```

Comment:
This looks wrong to me - `super().clean()` should update `self.cleaned_data` in place, and it doesn't generally return anything.